src/components/App/App.tsx

```diff
@@ -32,13 +32,14 @@ const App = () => {
   useEffect(() => {
     const allWeddings = async () => {
       const result = await getWeddings()
-      if(result.length > 0) {
+      console.log(result)
```

Comment:
This one was tricky. I always forget that strings have length too. Thanks for fixing that up